aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/deploying-web-packages.md

```diff
@@ -114,11 +114,54 @@ MSDeploy.exe relies on [Web Deploy providers](https://technet.microsoft.com/libr
 
 In addition, you'll need to specify various other [provider-specific settings](https://technet.microsoft.com/library/dd569001(WS.10).aspx) and general [operation settings](https://technet.microsoft.com/library/dd569089(WS.10).aspx). For example, suppose you want to deploy the ContactManager.Mvc web application to a staging environment. The deployment will target the Web Deploy Handler and must use basic authentication. To deploy the web application, you need to complete the next steps.
 
-**To deploy a web application using MSDeploy.exe**
+**To deploy a web application using MSDeploy.exe using an access token**
+
+MSDeploy V3 supports authentication with an access token, also known as a bearer token. If available, it is recommended to use an access token for best security.
+
+1. Build and package the web application project, as described in [Building and Packaging Web Application Projects](building-and-packaging-web-application-projects.md).
+1. Modify the *ContactManager.Mvc.SetParameters.xml* file to contain the correct parameter values for your staging environment, as described in [Configuring Parameters for Web Package Deployment](configuring-parameters-for-web-package-deployment.md).
+1. Open a Command Prompt window and browse to the location of MSDeploy.exe. This is typically at `%PROGRAMFILES%\IIS\Microsoft Web Deploy {version}\msdeploy.exe`.
+1. If you don't have an access token, create one using the command:
+
+   `az account get-access-token --query accessToken`
+
+1. Type this command, and then press Enter (disregard the line breaks):
+
+   ```cmd
+   MSDeploy.exe
+     -source:package="[path]\ContactManager.Mvc.zip"
+     -dest:auto,
+           computerName="https://stageweb1:8172/MSDeploy.axd?site=DemoSite",
+           username="FABRIKAM\stagingdeployer",
+           $CREDENTIAL_PLACEHOLDER$,
+           authtype="Bearer",
+           includeAcls="False",
+           Password="{token}"
+     -verb:sync
+     -disableLink:AppPoolExtension
+     -disableLink:ContentExtension
+     -disableLink:CertificateExtension
+     -setParamFile:"[path]\ContactManager.Mvc.SetParameters.xml"
+     -allowUntrusted
+     ```
+
+In this example:
+
+- The **–source** parameter specifies the **package** provider and indicates the location of the web package.
+- The **–dest** parameter specifies the **auto** provider. The **computerName** setting provides the service URL of the Web Deploy Handler on the destination server. The **authtype** setting `Bearer` indicates that you want to use an access token for authentication, and as such you need to provide the token value as the **password**. Finally, the **includeAcls="False"** setting indicates that you don't want to copy the access control lists (ACLs) of the files in your source web application to the destination server.
```

Comment:
Finally not needed and it's really not finally. You could explain them in any order. Less is more.

```suggestion
- The **–dest** parameter specifies the **auto** provider. The **computerName** setting provides the service URL of the Web Deploy Handler on the destination server. The **authtype** setting `Bearer` indicates that you want to use an access token for authentication, and as such you need to provide the token value as the **password**. The **includeAcls="False"** setting indicates that you don't want to copy the access control lists (ACLs) of the files in your source web application to the destination server.
```